internal/api/error.go

```diff
@@ -15,6 +15,7 @@ const (
 	entityTypeAction          entityType = "action"
 	entityTypeAlert           entityType = "alert"
 	entityTypeFilterAlert     entityType = "filter-alert"
+	entityTypeFeatureFlag     entityType = "feature-flag"
```

Comment:
Interesting. I didn't consider that we'd even add it here. LogScale itself doesn't really treat feature flags as "entities" as far as I'm aware. I wonder if we should consider retiring/removing the entire `internal/api/error.go` file soon, as it is mostly a left-over from the old API client that was located in https://github.com/humio/cli. We only really used this type to wrap our errors and make pretty output when printing errors to users of the `humioctl` tool.

That being said, this is in no way a blocker for this PR, mostly comments that we probably want to get rid of the entire file as it no longer serves the same purpose as it used to.

Reply:
Hm... makes sense, but should we keep these as "API error wrappers" ? I'm currently using it for and error when a feature flag is not found (said FeatureFlag name doesn't exist in LogScale).
I could also return a custom error for that. Do you think the latter is the right approach?